deployments/image/webtest/Dockerfile

```diff
@@ -15,23 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG ARCH=
-FROM ${ARCH}nginx:stable-alpine
-
-# Home location for all data and configs
-ENV HOME=/opt/yunikorn
+# Imagestage: use scratch base image
+FROM --platform=$TARGETPLATFORM scratch
+COPY --chown=4444:0 document/index.html /html/
```

Comment:
Use `--chown=0:0` here. There's no need to make the content owned by the web user.

Reply:
Looks good. Will +1 once we get clean e2e runs (may take a while as we've got some test flakiness).